project.py

```diff
@@ -336,29 +342,56 @@ def area_code_prefixes() -> list:
     return output_list
 
 
+def show_summary_data(title, data):
+    data_table = {title: data}
+    df = pd.DataFrame(data_table)
+    print(f'=== {title} ===')
+    # If there are more than 10 values show only the top-10
+    print(df.sort_values(by=title, ascending=False).head(10))

```

Comment:
To make the data for the pie chart easier to visually understand and use, I believe the data should be sorted. By moving the section ".sort_values(by=title, ascending=False)" in line 350 to the end of "pd.DataFrame(data_table)" in line 347, the variable df will already be sorted when it is initialized, ensuring that the table and pie chart are both sorted.

Reply:
Great comment.  The only issue is this globs together a lot of the text so I will implement this along with the solution to @Jmozebo93 comments